Sprint-3/reading-list/script.js

```diff
@@ -21,3 +21,26 @@ const books = [
   },
 ];
 
+let list = document.querySelector("#reading-list");
+
+books.forEach(item => {
+  let book = document.createElement('li')
+  book.innerHTML = `<div class="book-title">"${item.title}"</div>
+  <img src = ${item.bookCoverImage} alt= "book cover">
+  <button class= "btn">change status</button>`;
+  book.classList.add('book')
+  if (item.alreadyRead) {
```

Comment:
clever way of doing it with classes!
Well done!

Reply:
thank you !